angular/src/app/discovery/beacon/beacon-search.component.scss

```diff
@@ -80,6 +80,12 @@ ddap-beacon-search-result-table {
       width: 100%;
     }
   }
+
+  mat-drawer {
+    width: 100vw;
+    max-width: 100vw;
```

Comment:
According to the screenshot, I noticed two things:

1. If you look carefully, there is a border on the left of the drawer.
2. The total width of `mat-drawer` seem to push one extra pixel to the right as you can see in the screenshot from Safari.

> While `width` and `max-width` define the width of the [content](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#What_is_the_CSS_box_model) of `mat-drawer`, the total width after including the border is `100vw + 1px`.

You might want to remove the border.

Also, giving that the width is fixed, you might not need `max-width`.

Reply:
Good eye! I totally didn't see those at first